lib/auth/auth.go

```diff
@@ -105,7 +108,9 @@ func NewAuthServer(cfg *InitConfig, opts ...AuthServerOption) *AuthServer {
 		StaticTokens:                  cfg.StaticTokens,
 		ClusterAuthPreference:         cfg.ClusterAuthPreferenceService,
 		UniversalSecondFactorSettings: cfg.UniversalSecondFactorService,
+		trustedClusterBackend:         cfg.TrustedClusterBackend,
```

Comment:
why is this a private property while others are public? same about developermode